src/main/java/openblocks/common/entity/EntityHangGlider.java

```diff
@@ -49,10 +75,35 @@ public static void updateGliders(World worldObj) {
 		}
 	}
 
+	public static void toggleVario() {
+		if (!varioActive) {
+			varioActive = true;
+		} else {
+			varioActive = false;
+		}
+	}
+
+	public static void incVarioVol() {
+		varioVolume = Math.min((varioVolume + 2), VOL_MAX);
+		BeepGenerator.setVolume((byte) varioVolume);
+	}
+
+	public static void decVarioVol() {
+		varioVolume = Math.max((varioVolume - 2), VOL_MIN);
+		BeepGenerator.setVolume((byte) varioVolume);
+	}
+
 	private EntityPlayer player;
+	private PerlinNoiseGenerator noiseGen;
+	private BeepGenerator beeper;
+	private int ticksSinceLastVarioUpdate = 0;
+	private double avgVspeed = 0;
+	private double lastMotionY = 0;
 
 	public EntityHangGlider(World world) {
 		super(world);
+		this.noiseGen = new PerlinNoiseGenerator(world.getCurrentDate().get(GregorianCalendar.DAY_OF_YEAR));
```

Comment:
That's what I did at first, but it didn't work on multiplayer servers since you'd need OP rights.
I was also thinking about client-server communication to get the seed, but as of now the whole system would work completely without further network traffic.